.ci/run-tests

```diff
@@ -15,6 +15,12 @@ else
     export OPENSEARCH_URL_EXTENSION="http"
 fi
 
+export IS_UNRELEASED=false
+if [[ "$OPENSEARCH_VERSION" == *"SNAPSHOT" ]]; then
+  IS_UNRELEASED=true
```

Comment:
Where is this used?

Reply:
It is used in `run-opensearch` script to decide whether to get the image from docker, build and run it or use the image from unreleased OpenSearch assemble and run it.